ci/run_easymesh_cert.sh

```diff
@@ -54,29 +56,47 @@ is_prplmesh_device() {
     return 1
 }
 
+deploy_dummy_prplmesh() {
+    info "Deploy prplmesh locally to /opt/prplMesh"
+    rm -rf /opt/prplMesh
+    mkdir -p /opt/prplMesh
+    if ! mv "$rootdir/build/install" /opt/prplMesh ; then
+        err "Failed to deploy prplMesh locally, aborting"
+        exit 1
+    fi
+}
+
 upgrade_prplmesh() {
+    if [ "$TARGET_DEVICE" = prplmesh ] ; then
+        deploy_dummy_prplmesh && return 0 || return 1
+    fi
```

Comment:
I think it would be nicer to do:
```suggestion
    if [ "$TARGET_DEVICE" = prplmesh ] ; then
        deploy_dummy_prplmesh
    else    
        deploy_prplwrt_prplmesh
    fi
```
Note that the return is implied: the return value of the function is the one of the last statement. So if we have nothing after this, that is sufficient. But maybe a bit fragile.